scripts/qa_crowbarsetup.sh

```diff
@@ -2262,7 +2262,7 @@ function enable_ssl_generic()
             if [[ ! $(iscloudver 7plus) ]] ; then
                 return
             fi
-        ;;&
```

Comment:
@gosipyan this will change the logic though. We want to set `api` to `https` in case we are at cloud version 7plus, which is what the fall through `;;&` was trying to do. I don't know why this fails, maybe bash is too old on the ci node? In any case, please change line 2265 as you suggest, and also copy line 2267 into the `heat` switch.